### ISSUE TITLE
Missing data on Ryzen CPUs

### ISSUE THREAD

**Issue by workhorsy** [workhorsy/py-cpuinfo#115]
Looks like it is missing the CPU Hz, Family, Stepping, and Model.

It needs to parse CPU strings that donl't have @ or Hz
~~~
AMD Ryzen 7 2700X Eight-Core Processor
~~~

~~~
AMD Ryzen 7 2700X Eight-Core Processor          (3693.15-MHz K8-class CPU)
~~~

~~~
Intel(R) Pentium(R) CPU G640 @ 2.80GHz (family: 0x6, model: 0x2a, stepping: 0x7)
~~~
TrueOS 18
[system_info.txt](https://github.com/workhorsy/py-cpuinfo/files/2975967/system_info.txt)

**Comment by workhorsy** [comment]
open indiana
[system_info_openindiana.txt](https://github.com/workhorsy/py-cpuinfo/files/3001310/system_info_openindiana.txt)

**Comment by workhorsy** [comment]
OS X 10
[system_info_osx_10_14.txt](https://github.com/workhorsy/py-cpuinfo/files/3001393/system_info_osx_10_14.txt)
.14

**Comment by workhorsy** [comment]
Fixed in #121
